sync-core/src/main/java/com/cloudant/http/HttpConnectionManager.java

```diff
@@ -0,0 +1,47 @@
+package com.cloudant.http;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+/**
+ * Created by tomblench on 23/03/15.
+ */
+public class HttpConnectionManager {
+
```

Comment:
I'm not convinced we need this class. It doesn't add anything useful since we can just use `new HttpConnection` also the method names are also misleading since they don't connect to the remote unless I've missed something